db/migrate/20171117201519_update_resource_group_type_for_azure.rb

```diff
@@ -0,0 +1,9 @@
+class UpdateResourceGroupTypeForAzure < ActiveRecord::Migration[5.0]
+  class ResourceGroup < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  def change
```

Comment:
Will this work in a change?  I would expect this data migration only on up, and then on down, I guess it would get set back to `ResourceGroup`.